lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -47,7 +47,7 @@ Rails.application.config.after_initialize do
       api_key: ShopifyApp.configuration.api_key,
       api_secret_key: ShopifyApp.configuration.secret,
       api_version: ShopifyApp.configuration.api_version,
-      host: ENV['HOST'],
+      host: ENV.fetch('HOST'),
```

Comment:
Maybe this is only a 90% improvement because what does Sorbet regurgitate when it's an empty string? A good ol user-raised `ArgumentError` or `TypeError` with a nice message is best but, this is quick and simple.